library/core/src/cell.rs

```diff
@@ -1698,6 +1694,31 @@ impl<T: ?Sized> UnsafeCell<T> {
         // no guarantee for user code that this will work in future versions of the compiler!
         this as *const T as *mut T
     }
+
+    /// Returns a mutable reference to the underlying data.
+    ///
+    /// This call borrows the `UnsafeCell` mutably (at compile-time) which
+    /// guarantees that we possess the only reference.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(unsafe_cell_get_mut)]
+    /// use std::cell::UnsafeCell;
+    ///
+    /// let mut c = UnsafeCell::new(5);
+    /// *c.get_mut() += 1;
+    ///
+    /// assert_eq!(*c.get_mut(), 6);
+    /// ```
+    #[inline]
+    #[unstable(feature = "unsafe_cell_get_mut", issue = "none")]
```

Comment:
Note that I don't know the exact procedure stability-wise, btw:

  - (create an issue to attach the `#[unstable...` attribute to?)

     - (or could this be directly stabilized?)

 - (FCP?)

Reply:
Yes, please create a tracking issue (there's a template for that which you can pick when creating a new issue), and then put the issue's number here.

FCP will be needed for stabilization.